cluster-provision/k8s/1.18/provision.sh

```diff
@@ -204,13 +206,10 @@ spec:
 EOF
 
 
-default_cidr="192.168.0.0/16"
```

Comment:
Why did you drop this?

Reply:
see please the commit desc
https://github.com/kubevirt/kubevirtci/pull/422/commits/3a0b238baf22e64230238067b7a1cadc8f4bdafc

Reply:
Gotcha. Missed that